workshop/exercises/elastic/es-entrypoint.sh

```diff
@@ -35,7 +35,7 @@ set +e
 apt-get update -y &&
 apt-get install curl -y &&
 
-echo  "Waiting for ElasticSearch container..."
+echo  "Waiting for Elasticsearch container..."
 
 # First wait for ES to be up and then execute the original pygeoapi entrypoint.
-/wait-for-elasticsearch.sh http://elastic_search:9200 /entrypoint.sh || echo "ES failed: $?, exit" && exit 1
+/wait-for-elasticsearch.sh http://elasticsearch:9200 /entrypoint.sh || echo "Elasticsearch failed: $?, exit" && exit 1
```

Comment:
Ok now I see: the original dir was a complete copy of: 
https://github.com/geopython/pygeoapi/tree/master/docker/examples/elastic . So yes we can place a copy here under exercises/elastic, using the same structure, disadvantage is we need to maintain at two places.